src/components/App/App.test.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { shallow } from 'enzyme'
 import App from './App';
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
-});
+describe('App', () => {
+  it('should match the snapshot', () => {
+    const Header = () => {
+      return (
+        <header className='header'>
+          <h1 className='header-title'>Action Flix</h1>
+          <button className='login-btn'>Login</button>
+        </header>
+      )
+    }
+    const Route = jest.fn()
+    const movies = []
+
+    const wrapper = shallow(
+      <div className="App">
+        <Header />
+        <Route exact path="/" render={() => <MovieContainer movies={movies}/>}/>
+      </div>
+    )
+
+    expect(wrapper).toMatchSnapshot();
+  });
+})
+
+
```

Comment:
Great job on this Joel! For these test we don't actually need to use all the ui elements. You can just assign `wrapper` to `shallow(<App />)'  and that mocks the render of the component and then all we have to do is check that wrapper then matches the snapshot.

Reply:
ah cool, I didn't realize that! Thanks Derek